boards/MicroBit/src/microbit-i2c.adb

```diff
@@ -52,6 +52,14 @@ package body MicroBit.I2C is
    ----------------
 
    procedure Initialize (S : Speed := S400kbps) is
+      Config   : GPIO_Configuration := (Mode => Mode_In,
+                                        Resistors => Pull_Up,
+                                        Input_Buffer => Input_Buffer_Connect,
+                                        Drive => Drive_S0D1,
+                                        Sense => Sense_Disabled);
+
+      GPIO_Pin_SCL : GPIO_Point := (Pin => MB_SCL.Pin);
+      GPIO_Pin_SDA : GPIO_Point := (Pin => MB_SDA.Pin);
```

Comment:
This variables are not needed. You can directly use `MB_SCL` and `MB_SDA` below.